tests/test_cli.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from typer.testing import CliRunner
+
+from gtfs.__main__ import app
+from gtfs.utils.constants import *
+from gtfs.utils.geom import bbox_contains_bbox, bbox_intersects_bbox
+
+
+@pytest.fixture(scope="module")
+def runner():
+    return CliRunner()
+
+
+class Test_Geom_Functions:
+    @pytest.mark.parametrize(
+        "bbox1, bbox2",
+        [bbox_covers_bbox, bbox_contained_inside_bbox],
```

Comment:
I think it's easier to read if we avoid variables here and rather repeat coordinates here, so it's easier to reason about the test. best to format the coordinates well (might have to turn off `black` for that) for the extra readability :D

Reply:
I was wondering what kind of formatting we could do since there should be a way to define these input coords like maybe at least add comments....also for some tests there are many permutations possible so should we just keep a few?

Reply:
I have made a new commit you can see those changes and then tell me what should be done according to u...